charts/nri-bundle/README.md.gotmpl

```diff
@@ -97,10 +97,11 @@ newrelic-infrastructure:
 ## Bring your own KSM
 
 New Relic Kubernetes Integration requires an instance of kube-state-metrics (KSM) to be running in the cluster, which this chart pulls as a dependency. If you are already running or want to run your own KSM instance, you will need to make some small adjustments as described below.
+If you are running Kubernetes Integration up to version v2.13.5, note that it is only compatible with KSM v1.9.8. In that case you can run KSM v1.9.8 alongside your own KSM following [these steps](#ksm198).
```

Comment:
The V2 branch is maintained [here](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle-legacy). I do not think it is needed to add a callout here